components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -760,6 +760,7 @@ protected void findPreviousAuthenticatedSession(HttpServletRequest request, Auth
                     // authentication step updates will be reflected.
                     refreshAppConfig(effectiveSequence, request.getParameter(FrameworkConstants.RequestParams.ISSUER),
                             context.getRequestType(), context.getTenantDomain());
+                    context.setAuthenticatedIdPsPerApp(sessionContext.getAuthenticatedIdPsPerApp().get(appName));
```

Comment:
So this `AuthenticatedIdPsPerApp` for the application? then the name should be `AuthenticatedIdPsOfApp` or something like that ?

Reply:
yes. This is for application

Reply:
fixed it